azuredeploy.json

```diff
@@ -1,19 +1,23 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.1",
     "parameters": {
         "scopeID": {
-            "type": "String"
+            "type": "string"
         },
         "iotCentralSASKey": {
-            "type": "String"
+            "type": "string"
+        },
+        "currentTime": {
+            "type": "string",
+            "defaultValue": "[utcNow('u')]"
```

Comment:
I want to create unique name for each Azure resource so that every deployment wouldn't overwrite previous deployment in the same resource group.

Reply:
I understand. But the problem here is that this will show up in the UI in the parameter section with this unparsed default value, which makes it confusing for the user. I wouldn't touch this flow, since we are not changing functionality, just upgrading the version.

Reply:
Yeah I see that.